src/fiber/fiber.ml

```diff
@@ -361,6 +361,22 @@ let parallel_map l ~f k =
 
 let all_concurrently = parallel_map ~f:Fun.id
 
+let all_concurrently_unit (l : unit t list) k =
```

Comment:
`all` is implemented in terms of `sequential_map` and `all_concurrently` is implemented in terms of `parallel_map`, so I would expect `all_concurrently_unit` to be implemented in terms of `paraller_iter`.

Reply:
Done.